tests/ctags/simple.d.tags

```diff
@@ -20,5 +20,5 @@ obj
 qar�64�Struct.Union�0�int
 quxx�64�Struct.Union�0�bool
 test.simple�256�0
-tfun�16�(T)�Class�0�auto
+tfun�16�(T v)�Class�0�auto
```

Comment:
This doesn't seem to be a valid D code (or valid code for any function-like declaration from c.c). The way signature collection works now it probably replaces the original signature "T" with the second signature "T v" when the second "(" appears. I could probably adjust the signature collection code but not sure if it's worth it as this shouldn't happen in a valid code.